### PR TITLE
[DOCS-4907] Update APM and Logs PCI docs

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do?

Updates the APM and Logs PCI docs with Agent config example. Also add more details to PCI section for APM.
Ready to merge.

### Motivation

[DOCS-4907](https://datadoghq.atlassian.net/browse/DOCS-4907)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-4907]: https://datadoghq.atlassian.net/browse/DOCS-4907?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by le-he-hoo — APPROVED** [review]

File: content/en/tracing/configure_data_security/_index.md [hunk 1]
```diff
@@ -342,18 +342,33 @@ Datadog may gather environmental and diagnostic information about your tracing l
...
+PCI compliance for APM is available when you create a new Datadog organization. To set up a PCI-compliant Datadog org, follow these steps:
 
-[1]: /account_management/audit_trail/
+1. Set up a new Datadog org in the [US1 site][1]. PCI DSS compliance is only supported for new orgs created in US1.
+2. Contact [Datadog support][2] or your [Customer Success Manager][3] to request that the new org be configured as a PCI-compliant org.
+3. Enable [Audit Trail][4] in the new org. Audit Trail must be enabled and remain enabled for PCI DSS compliance.
+4. After Datadog support or Customer Success confirms that the new org is PCI DSS compliant, configure the Agent configuration file to send spans to the dedicated PCI-compliant endpoint (`https://trace-pci.agent.datadoghq.com`):
+    ```
+    apm_config:
+    apm_dd_url: <https://trace-pci.agent.datadoghq.com>
```
Comment (le-he-hoo) [inline content/en/tracing/configure_data_security/_index.md:361]: This line should be indented 2 spaces so that it is nested under `apm_config` 👍

Comment (maycmlee) [reply content/en/tracing/configure_data_security/_index.md]: Good catch! Fixed.

File: content/en/data_security/logs.md [hunk 2]
```diff
@@ -54,23 +54,30 @@ If you have any questions about how the Log Management Service satisfies the app
...
+4. After Datadog support or Customer Success confirms that the new org is PCI DSS compliant, configure the Agent configuration file to send logs to the dedicated PCI-compliant endpoint (`agent-http-intake-pci.logs.datadoghq.com`):
+    ```
+    logs_config:
+      logs_dd_url: <http://agent-http-intake-pci.logs.datadoghq.com:443|agent-http-intake-pci.logs.datadoghq.com:443>
+    ```
+    **Note**: The port must be included in the configuration.
 
 If you have any questions about how the Log Management service satisfies the applicable requirements under PCI DSS, contact your account manager.
 
+To enable PCI compliance for APM, see [PCI DSS compliance for APM][5]
```
Comment (drichards-87) [inline content/en/data_security/logs.md:74]:
```suggestion
To enable PCI compliance for APM, see [PCI DSS compliance for APM][5].
```

**Review by drichards-87 — APPROVED** [review]
Looks good. I only had one minor piece of feedback.

**Pull request merged**